<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="comboBox_categories">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Все задачи</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_task">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	padding-left: 20px;
	padding-right: 20px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: #666;
	color: white;
	transition: transform 0.2s;
}

QPushButton:hover {
  cursor: pointer;
  background-color: gray;
}

QPushButton:pressed {
  transform: scale(0.5);
  background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_category">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Категория:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/addTaskWindow/trash.png</normaloff>../icons/addTaskWindow/trash.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(33, 33, 33);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>comboBox_categories</zorder>
  <zorder>label_date</zorder>
  <zorder>lineEdit_task</zorder>
  <zorder>btn_add_edit</zorder>
  <zorder>label_category</zorder>
  <zorder>btn_delete</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
